defaults/main.yml

```diff
@@ -2,3 +2,5 @@ podman_user_name: osbs-podman
 podman_user_uid: 2022
 podman_user_group_name: osbs-podman
 podman_user_group_gid: 2022
+podman_pruning_cmd: 'podman system prune --all --filter "until=10m" -f'
```

Comment:
Can we have instead of the whole command, just `until` time limit?

Reply:
sure